scripts/release/bug_bash.py

```diff
@@ -26,39 +26,39 @@ def parse_args():
     parser.add_argument('-a', '--all', dest='all',
                         help='Show all bugs squashed', action='store_true')
     parser.add_argument('-t', '--token', dest='tokenfile',
-                        help='File containing GitHub token', metavar='FILE')
-    parser.add_argument('-b', '--begin', dest='begin', help='begin date (YYYY-mm-dd)',
+                        help='File containing GitHub token (altnernatively, use GH_TOKEN env variable)', metavar='FILE')
+    parser.add_argument('-s', '--start', dest='start', help='start date (YYYY-mm-dd)',
                         metavar='date', type=valid_date_type, required=True)
     parser.add_argument('-e', '--end', dest='end', help='end date (YYYY-mm-dd)',
                         metavar='date', type=valid_date_type, required=True)
 
     args = parser.parse_args()
 
-    if args.end < args.begin:
+    if args.end < args.start:
         raise ValueError(
-            'end date {} is before begin date {}'.format(args.end, args.begin))
+            'end date {} is before start date {}'.format(args.end, args.start))
 
     if args.tokenfile:
         with open(args.tokenfile, 'r') as file:
             token = file.read()
             token = token.strip()
     else:
-        if 'GITHUB_TOKEN' not in os.environ:
+        if 'GH_TOKEN' not in os.environ:
             raise ValueError('No credentials specified')
-        token = os.environ['GITHUB_TOKEN']
+        token = os.environ['GH_TOKEN']
```

Comment:
Any reason behind this change? FWIW, GitHub calls it `GITHUB_TOKEN` (`${{ secrets.GITHUB_TOKEN }}`) and there are other scripts that use this name as well (see https://github.com/zephyrproject-rtos/zephyr/pull/45540).

Reply:
Ah weird - I thought I saw another script that used this convention. Have not been doing ci much lately

Reply:
https://github.com/zephyrproject-rtos/zephyr/blob/21d1ad3762302b3e461953df59430c77e0709274/scripts/release/list_issues.py#L150

right, there is `list_issues.py` that uses `GH_TOKEN`. Could you instead update this file to use `GITHUB_TOKEN` here?